src/Model/Callback/PassThreadControl.php

```diff
@@ -19,12 +19,12 @@ class PassThreadControl
     /**
      * TakeThreadControl constructor.
      *
-     * @param string $newOwnerAppId
-     * @param string $metadata
+     * @param $newOwnerAppId
+     * @param string|null $metadata
      */
-    public function __construct(string $newOwnerAppId, string $metadata)
+    public function __construct($newOwnerAppId, ?string $metadata)
```

Comment:
What's the type of $newOwnerAppId ? If it's an int that is always return then in this case, can you add int for type hinting instead of nothing, removing the casting into a string and change the return type of the getter please ?